lib/bundler/cli/plugin.rb

```diff
@@ -5,14 +5,22 @@ module Bundler
   class CLI::Plugin < Thor
     desc "install PLUGINS", "Install the plugin from the source"
     long_desc <<-D
-      Install plugins either from the rubygems source provided (with --source option) or from a git source provided with (--git option). If no sources are provided, it uses Gem.sources
+      Install plugins from any of the sources provided:
+
+      - from rubygems (`--source` option)
+      - from a remote git source (`--git` option)
+      - from a local git repo (`--file /path/to/repo` option)
+
+      If no sources are provided, it uses Gem.sources.
    D
     method_option "source", :type => :string, :default => nil, :banner =>
       "URL of the RubyGems source to fetch the plugin from"
     method_option "version", :type => :string, :default => nil, :banner =>
       "The version of the plugin to fetch"
     method_option "git", :type => :string, :default => nil, :banner =>
       "URL of the git repo to fetch from"
+    method_option "file", :type => :string, :default => nil, :banner =>
+      "Path of the local git repo to fetch from"
```

Comment:
I think it's confusing we use `--file` to point to a local git repo instead of just a path on disk, maybe call it something like `--local-git` ?